### PR TITLE
i18n: FR translation update (8)

### PR DESCRIPTION
**Pull request by mooondark**

### CODE REVIEW

File: ui/raidboss/data/06-ew/trial/golbez-ex.ts [hunk 1]
```diff
@@ -382,6 +390,7 @@ const triggerSet: TriggerSet<Data> = {
         text: {
           en: '(in + spread, for later)',
           de: '(rein + verteilen, für später)',
+          fr: '(intérieur + écartement, pour après',
```
Comment (wexxlee) [inline ui/raidboss/data/06-ew/trial/golbez-ex.ts:393]: Might be missing the close parenthesis? ")"

File: ui/raidboss/data/06-ew/trial/zodiark-ex.ts [hunk 2]
```diff
@@ -118,6 +118,7 @@ const paradeigmaLeanOutputStrings = {
   dirWithLean: {
     en: '${dir} bird (lean ${lean})',
     de: '${dir} Vogel (nach ${lean} neigen)',
+    fr: 'Oiseau ${dir}(côté ${lean})',
```
Comment (wexxlee) [inline ui/raidboss/data/06-ew/trial/zodiark-ex.ts:121]: Might need a space between "${dir}(côté..." ?

File: ui/raidboss/data/06-ew/trial/golbez-ex.ts (hunk 1 above)
Comment (mooondark) [reply ui/raidboss/data/06-ew/trial/golbez-ex.ts]: Good catch 👍

**Review by wexxlee — APPROVED** [review]
Thanks for all the i18n work!

**Pull request merged**